_schemaV2.graphql

```diff
@@ -9200,6 +9201,7 @@ type Query {
   # Find PartnerStats
   analyticsPartnerStats(partnerId: String!): AnalyticsPartnerStats
   marketingCollections(
+    category: String
```

Comment:
This change seems unrelated to the order count field. Is it supposed to be included in this PR?